### PR TITLE
Hotfix/asteroids

### PR DESCRIPTION
**Pull request by GitGitarra**

### CODE REVIEW

File: VatSpace/main.lua [hunk 1]
```diff
@@ -36,7 +36,14 @@ end
 
 local function restartGameForNextLevel()
     if gameTimer then
-        stopTimer()        
+        stopTimer() 
+        asteroids.destroy()
+        asteroids.create()
```
Comment (s-kania) [inline VatSpace/main.lua]: Hmm, what do you think about create method recreate() in asteroids to encapsulate these two functions?

File: VatSpace/main.lua [hunk 2]
```diff
@@ -36,7 +36,14 @@ end
...
+        cats.cat:toBack()  
+        ui.goalTextGroup:toFront()
+        ui.morawieckiGroup:toFront()
+        ui.progressViewGroup:toFront()
```
Comment (s-kania) [inline VatSpace/main.lua]: Cool methods, but as I see in documentation, its moving objects only in their groups. Are you sure that is working for our problem?

Comment (GitGitarra) [reply VatSpace/main.lua]: I read documentation :) And it's work fine in our case (except for cat but this is separate topic).

Comment (GitGitarra) [reply VatSpace/main.lua]: What do you think about encapsulate this 3 functions to new function? I think i'ts even more needed then recreate() function you mention before.

Comment (s-kania) [reply VatSpace/main.lua]: Definitely, it should't be in main.

File: VatSpace/main.lua (hunk 1 above)
Comment (GitGitarra) [reply VatSpace/main.lua]: Done.

File: VatSpace/main.lua (hunk 2 above)
Comment (GitGitarra) [reply VatSpace/main.lua]: Done.

**Pull request merged**